source/Octopus.Tentacle/Kubernetes/KubernetesConfigMapService.cs

```diff
@@ -0,0 +1,49 @@
+using System.Collections.Generic;
+using System.Net;
+using System.Threading;
+using System.Threading.Tasks;
+using k8s;
+using k8s.Autorest;
+using k8s.Models;
+
+namespace Octopus.Tentacle.Kubernetes
+{
+    public interface IKubernetesV1ConfigMapService
+    {
+        Task<V1ConfigMap?> TryGet(string name, CancellationToken cancellationToken);
+
+        Task<V1ConfigMap> Patch(string name, IDictionary<string, string> data, CancellationToken cancellationToken);
+    }
+
+    public class KubernetesV1ConfigMapService : KubernetesService, IKubernetesV1ConfigMapService
```

Comment:
You can drop the `V1` here. I doubt we'll need to worry about `V2` K8s apis/types for a while

Reply:
Removed 👍